src/components/Hero/Hero.tsx

```diff
@@ -120,6 +117,23 @@ export const Hero: ChakraComponent<
       let backgroundImageStyle = {};
       let contentBoxStyling = {};
 
+      const imageToRender = (
+        <Image
+          alt={imageProps.alt}
+          component={imageProps.component}
+          fallbackSrc={imageProps.fallbackSrc}
+          id={imageProps.id}
+          onError={imageProps.onError}
+          src={imageProps.src}
+        />
+      );
+
+      // Using the custom Image component's props to check requirements.
+      if (imageProps.component) {
+        const { src, alt } = imageProps.component.props;
+        Object.assign(imageProps, { src, alt });
+      }
```

Comment:
Regarding your first note in the PR description, it is confusing and you can use the default behavior for the `component` prop in the `Image` component. It should be so that what you pass as `component` is used so you don't need the other props in `imageProps`.

In the story example
```
const componentImageProps = {
  component: (
    <Image alt={"Custom Image component example"} src={getPlaceholderImage()} />
  ),
};
```

This should propagate to the `Image` component so assigning the prop values to `imageProps` should not be necessary.

Reply:
The test you wrote passed when I commented this out.

Reply:
I think that's fine. We should log the warnings if the props we support are not passed in. But if the user passes in their own image component, they're responsible for setting it correctly.